### ISSUE TITLE
Add function(s) for creating Gobi lists for Data Export

### ISSUE THREAD

**Issue by jermnelson** [sul-dlss/libsys-airflow#872]
Gobi requires the generation of ISBN lists specified [here](https://docs.google.com/document/d/1NAbY9L6G16DItIBjtRQKFDcL6NN5cWK8VfsFCT48vTA/edit#heading=h.ug2nnjszleyr) and does not require MARC processing or transmission.

Send a text file with 3 pipe-delimited fields on each line. The fields in order are the ISBN, "print" or "ebook", and account number "325099"
_Note: some ISBNs have two lines in the export, one each for "print" or "ebook" if the record is considered both print and ebook._

568987545678|print|325099
987654567890|ebook|325099
675456787678|print|325099
675456787678|ebook|325099
…etc

File Naming: stf.{%Y%m%d}.txt

A record is considered print if it has an item.
For those in ebook list:
856/956: NO subfield ‘x’ text include ”subscribed” AND only has holdings record(s) type=electronic
The conditions for both print and ebook could be true, in which case the record would be considered both print and ebook.

**Comment by ahafele** [comment]
@shelleydoljack regarding the request for an isbn/eisbn spec, I don't think we are currently doing that. Looking at /s/SUL/Bin/ExportHistory/ybp_coutts_export.pl I don't see anything that would select based on text electronic bk. (e.g. 020: 0702054380 ǂq (electronic bk.)) in the 020. I think maybe we are just sending all 020$a. It does look like we are tossing invalid isbns and doing some deduplication between each list.

**Comment by ahafele** [comment]
Looking at old report and we were validating isbns so I assume we need to continue doing that. 

```
01:43:12  Not a valid ISBN for ckey 14903118 in 020 subfield a: '898700751H'
01:43:17  Not a valid ISBN for ckey 15072573 in 020 subfield a: '9781760623531.00000033'
01:43:18  Not a valid ISBN for ckey 15076307 in 020 subfield a: 'IN00207420'
01:43:19  Not a valid ISBN for ckey 14892459 in 020 subfield a: '898657151E'
01:43:19  Not a valid ISBN for ckey 14892597 in 020 subfield a: '898726160F'
01:43:19  Not a valid ISBN for ckey 14892704 in 020 subfield a: '876510300E'
01:43:19  Not a valid ISBN for ckey 14892729 in 020 subfield a: '898565182I'
```

**Comment by shelleydoljack** [comment]
Doesn't folio provide some ISBN validation as part of orders? Maybe we can use that?

**Comment by jgreben** [comment]
Closed by https://github.com/sul-dlss/libsys-airflow/pull/937

**Comment by ahafele** [comment]
@jgreben Suzette confirms that we want to continue to send the ebook/print designation. This data is used in the GOBI ordering system by staff for dedupping workflows. 
I found the official spec document from GOBI here https://connect.ebsco.com/s/article/GOBI-Library-Holdings-Load-Service?language=en_US which includes some additional isbn information, for example 
> We do not need the ISBN for both the cloth (hardcover) and the paperback bindings of a title. We link different binding editions of a title in GOBI, so if you send the cloth and paper ISBN, and we have both in our system and linked, “already owned” and “alt-ed already owned” will appear on each record.
What we were previously doing can be found in /s/SUL/Bin/ExportHistory/ybp_coutts_export.pl

**Comment by jgreben** [comment]
Fixed by #965

**Comment by ahafele** [comment]
> Fixed by #965

I think this comment is on wrong ticket?

**Comment by jgreben** [comment]
Fixed by https://github.com/sul-dlss/libsys-airflow/pull/937
